editors/code/src/installation/server.ts

```diff
@@ -89,8 +89,8 @@ async function downloadServer(source: BinarySource.GithubRelease): Promise<boole
     }
 
     const binaryPath = path.join(source.dir, source.file);
-    
-    if (!isBinaryAvailable(binaryPath)) assert(false,
```

Comment:
I put the if there to evaluate the error message lazily, though it may not be that important...

Reply:
Yeah, it's not important